isis/src/tgo/apps/tgocassismos/main.cpp

```diff
@@ -184,17 +186,27 @@ void IsisMain() {
         observationId =  archiveGroup["ObservationId"][0];
         startTime =      instGroup["StartTime"][0];
         startClock =     instGroup["SpacecraftClockStartCount"][0];
+        stopTime   =     instGroup["StartTime"][0]; 
```

Comment:
Is this meant to be "StopTime" by chance?

Reply:
Actually, no! Stop time doesn't exist in the incoming label, so we take the last StartTime and add the exposure duration to get the StopTime.

Reply:
Ah - makes sense!